esp/src/eclwatch/LockDialogWidget.js

```diff
@@ -0,0 +1,122 @@
+/*##############################################################################
+#   HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
```

Comment:
I believe new source files get the current year specified here

Reply:
@richardkchapman Any input on this? Should we add 2018 HPCC Systems here?

Reply:
Yes

Reply:
Will modify @richardkchapman